.github/workflows/release.yml

```diff
@@ -122,4 +138,30 @@ jobs:
           sonatype_pwd: ${{ secrets.SONATYPE_NEXUS_PASSWORD }}
           signing_key_id: ${{ secrets.SIGNING_KEY_ID }}
           signing_key_pwd: ${{ secrets.SIGNING_KEY_PASSWORD }}
+          signing_key_file: ${{ env.SIGNING_KEY_FILE_PATH }}
```

Comment:
Have these all been updated to a generic PayPal user (not anyone's individual username and password?)

Reply:
I personally think that should be a part of this ticket. We don't want to be releasing under individual sonatype accounts.